src/components/PlanPage/PlanPage.js

```diff
@@ -9,14 +9,10 @@ import { startSetActivities } from '../../store/activities/actions';
 const PlanPage = ({ onStartSetActivities }) => {
   const [isDataLoaded, setIsDataLoaded] = useState(false);
 
-  const asyncOnStartSetActivities = () => {
-    (async () => {
-      await onStartSetActivities();
-      setIsDataLoaded(true);
-    })();
-  };
-
-  useEffect(asyncOnStartSetActivities, []);
+  useEffect(async () => {
```

Comment:
we cannot do like that. React docs says, that we cannot pass async function to `useEffect` - go and read the whole React docs about hooks (btw, you should read as much as possible of react docs). If it works right now, it can fail if you will have more components. 

you have to define function inside of `useEffect`. 

So: 
```
useEffect(() => {
  const yourFuckingFunction =  async () => {
    await onStartSetActivities();
    setIsDataLoaded(true);
 };

  yourFuckingFunction();
}, [])
```